apps/microsoft-teams/frontend/src/components/config/AccessSection/AccessSection.styles.ts

```diff
@@ -7,4 +7,8 @@ export const styles = {
     width: 16px;
   }
 `),
+  orgLogo: css({
+    /** Setting size to default size for favicon in MS organizational branding */
+    height: 32,
```

Comment:
this is more of a general question, but curious if it makes sense for us to start using tokens for sizing values? Can't remember where I saw that suggested recently, but what are your thoughts?

Reply:
Yes, I've definitely used the [Forma36 tokens](https://f36.contentful.com/tokens/color-system), and we should try to use those whenever possible. They only have spacing values or sizing on specific Forma components, so I'm not sure it's the best use case here unless I'm missing something!